swap/swap.go

```diff
@@ -655,12 +659,13 @@ func (s *Swap) Deploy(ctx context.Context, initialDepositAmount uint64) (contrac
 	// initial topup value
 	opts.Value = big.NewInt(int64(initialDepositAmount))
 	opts.Context = ctx
-	swapLog.Info("Deploying new swap", "owner", opts.From.Hex(), "deposit", opts.Value)
+	swapLog.Info("Deploying new swap", "owner", opts.From.Hex(), "deposit", opts.Value, "action", "deploy")
```

Comment:
There are intertwined, I think it's best if `swapLog.SetLogAction("swap_deploy_contract")` is placed before `"Deploy new swap"`

> ...which contract we're talking about, since we're about to have more than 1...

I'm not sure how can we distinguish them, is there a field that can be used for this purpose.

Reply:
of course, 1 action for both of these functions seems OK to me :+1: 

regarding distinguishing this contract from others: this one is a chequebook contract—it would be different from, say, an oracle contract.

besides that: remove `swap` from the set action

Reply:
Fixed